### PR TITLE
Feature – Add additional accounts

### LINKED ISSUE #105: Integration iov-core to allow multi account creation
**Description**
For finishing completely the account status view, we need to give support for multi-account creation inside persona.

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #105

### CODE REVIEW

File: packages/sanes-chrome-extension/src/extension/background/actions/createAccount/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { CreateAccountResponse } from '../../messages';
+import { getCreatedPersona } from '../createPersona';
+
+export async function createAccount(): Promise<CreateAccountResponse> {
```
Comment (apanizo) [inline packages/sanes-chrome-extension/src/extension/background/actions/createAccount/index.ts]: I am going to propose, for reducing complexity migrate all communication between the extension -> bacgkround script using the same approach I implemented for getting the requests' queue. As you have already identified, `getBackgroundPage`. 

In this way, we save some back and forth of messages. But we can do it easily in a separate PR. Not urgent now.

File: packages/sanes-chrome-extension/src/logic/persona/persona.unit.spec.ts [hunk 2]
```diff
@@ -58,6 +58,28 @@ withChainsDescribe('Persona', () => {
     });
   });
 
+  describe('createAccount', () => {
+    it('can create an account', async () => {
+      const persona = await Persona.create();
+
+      {
```
[the remark this answers is not in the document]
Comment (webmaster128) [reply packages/sanes-chrome-extension/src/logic/persona/persona.unit.spec.ts]: It creates a new scope, such that scoped variables (`let`, `const`) are only available in this scope. E.g.

```js
const a = 1;

// a exists here

{
  const b = 2;
  // a exists here, b exists here
}

// a axists here, but no b anymore, since b went out of scope.
```

This is useful in tests when you want to reuse the same variable names multiple times. Here I have 2 `const accounts`.

File: packages/sanes-chrome-extension/src/extension/background/actions/createAccount/index.ts (hunk 1 above)
Comment (webmaster128) [reply packages/sanes-chrome-extension/src/extension/background/actions/createAccount/index.ts]: I agree, let's hope this works nicely. This also removes the limitation of JSON compatible data
